prototype/water-extent-map.ipynb

```diff
@@ -0,0 +1,242 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# HyP3 prototype water extent map product\n",
+    "\n",
+    "This notebook mimics the workflow a HyP3 Water Map plugin will follow to create an On Demand surface water extent map. It is intended demonstrate *how* HyP3 goes from an On Demand request to a product as well as provide a platform for testing/tuning/refinement ot the water extent map product.\n",
+    "\n",
+    "Importantly, step 3 will become the plugin's primary inteface and the options provided to the `make_water_map` function will be the ones provided to users (or a subset thereof). Steps 1 and 2 are required data marshelling steps that will be handled transparently inside of the container.\n",
```

Comment:
not sure if `or a subset thereof` means a subset of options or a subset of users?